app/controllers/task_controller.rb

```diff
@@ -14,6 +14,20 @@ def show
 		render :details
```

Comment:
By the way, it's customary to use the plural of the model class name in the controller name as you did with `Person` and `PeopleController`. Here you have a `TaskController` rather than `TasksController`. Due to some of the built-in magic in rails routing, this used to cause lots of headaches when you didn't follow the pattern. I think that's not quite as much of an issue anymore, and clearly it seems you've got it working.